README.md

```diff
@@ -1,2 +1,12 @@
 # inji-vc-renderer
-Description: Kotlin and JS library to render VC with SVG template support
+This repository contains Kotlin and JS library to render VC with SVG template support.
+
+### Kotlin
+- Kotlin directory has the library which generates aar and jar for VC Rendering.
+- It also has the example application which uses the library code.
+- More information on the  [Kotlin library Read Me](kotlin/Readme.md)
```

Comment:
More information on the Kotlin library [check here](kotlin/Readme.md)

Reply:
Done